async-query-core/src/main/java/org/opensearch/sql/spark/validator/SQLQueryValidationVisitor.java

```diff
@@ -561,12 +561,18 @@ private void validateFunctionAllowed(String function) {
       case MAP:
         validateAllowed(GrammarElement.MAP_FUNCTIONS);
         break;
+      case BITWISE:
```

Comment:
so we need to keep adding case and there is no way to convert `function` to grammar element?

Reply:
Let me go ahead with the current approach for now, I found some test case conflict if I enable validation for all the function types.